packages/metrics_center/lib/src/constants.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Strings used for GIT tags
```

Comment:
It's probably more accurate to have `GIT tags` -> `[MetricPoint] tags`

Reply:
Done.

A lot of these docs are ... sparse. But it's better than nothing at all I guess.